test/Integration/RequestTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-diactoros for the canonical source repository
+ * @copyright Copyright (c) 2015-2017 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
Change this line in all new files to:

```php
 * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
```

(Copyright extends from when the _file_ is introduced. Also, updates to use SSL-enabled URL.)